bwapi/BWAPILIB/BWAPILIB.vcxproj

```diff
@@ -238,7 +239,17 @@ COPY /y "$(OutDir)BWAPILibd.pdb" "..\lib\BWAPILibd.pdb"
     <ClInclude Include="..\include\BWAPI\UpgradeType.h" />
     <ClInclude Include="..\include\BWAPI\WeaponType.h" />
   </ItemGroup>
+  <ItemGroup>

```

Comment:
Please remove everything you added here and below. Nuget automatically reads `packages.config` and doesn't need this.